avocado_vt/loader.py

```diff
@@ -188,9 +199,7 @@ def discover(self, url, which_tests=loader.DiscoverMode.DEFAULT):
         for params in (_ for _ in cartesian_parser.get_dicts()):
             # Evaluate the proper avocado-vt test name
             test_name = None
-            if get_opt(self.config, 'vt.config'):
-                test_name = params.get("shortname")
```

Comment:
Sometimes, we will define different test matrices in the configuration file, such as different guests, different drivers, so they will test the same test case. Using shortname can help us analyze the testing status of each test case. So I think this change is open to discussion. Cc @luckyh @yanan-fu 

```
_short_name_map_file = {'host.cfg': 'Host.RHEL.m8.u3.Host_arch_aarch64', 'machines.cfg': 'arm64-pci', 'subtests.cfg': 'io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads', 'guest-os.cfg': 'Guest.Linux.RHEL.8.3.0.aarch64', 'guest-hw.cfg': 'bridge.default_bios.no_virtio_rng.filesystem.no_9p_export.smallpages.no_pci_assignable.qcow2.virtio_scsi.up.virtio_net', 'tests-example.cfg': 'pre_test'}

shortname = qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.0.aarch64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.arm64-pci
id = io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads
```